src/FrankenPhp/Concerns/FindsFrankenPhpBinary.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Laravel\Octane\FrankenPhp\Concerns;
+
+use Illuminate\Support\Str;
+use Symfony\Component\Process\ExecutableFinder;
+
+trait FindsFrankenPhpBinary
+{
+    /**
+     * Find the FrankenPHP binary used by the application.
+     */
+    protected function findFrankenPhpBinary(): ?string
+    {
+        if (file_exists($basePath = base_path('frankenphp'))) {
+            return $basePath;
+        }
```

Comment:
This part makes me wonder if we should check whether it's actually a file and an executable one, like this:

```suggestion
        $frankenPhpBinary = base_path('frankenphp');

        if (is_file($frankenPhpBinary) && is_executable($frankenPhpBinary)) {
            return $frankenPhpBinary;
        }
```

Because `file_exists()` could be anything, like an empty text file or a binary without execution privileges or even a folder called `/frankenphp`, and so on.

Maybe I'm being too strict here, so, you can just ignore this. =D

Reply:
Actually, this block is useless (it was a copy/paste from the RoadRunner installer, which also contains useless code). Fixed.

Reply:
@dunglas I think the idea behind the old code was: "Let's prioritize finding the binary in base_path. As a last resort, we'll try to find it in the `$PATH`." By doing this: `(new ExecutableFinder())->find('frankenphp', null, [base_path()]);`, I'm pretty confident that it will first try to find the binary in various $PATH and finally in base_path as the last one. 

But I don't think this is an issue, right? I think it's okay. =)